test/crossbrowser/src/browser/api.ts

```diff
@@ -0,0 +1,362 @@
+import {
+  ResourceID,
+  add,
+  lookup,
+  remove
+} from './resources';
+
+import {
+  serializeLocalParticipant,
+  serializeLocalTrack,
+  serializeLocalTrackPublication,
+  serializeRoom
+} from './serialize';
+
+declare const Twilio: any;
+
+/**
+ * Connect to a {@link Room}.
+ * @param {Array<*>} args
+ * @param {(room: Room) => void} sendRoomEvents
+ * @returns {Promise<object>}
+ */
+export async function connect(args: any, sendRoomEvents: (room: any) => void): Promise<any> {
+  const [ token, options ] = args;
+  let room: any;
+
+  try {
+    room = await Twilio.Video.connect(token, options);
+  } catch (e) {
+    return {
+      error: {
+        code: e.code,
+        message: e.message
+      }
+    };
+  }
+
+  add(room);
+  add(room.localParticipant);
+  room.localParticipant.tracks.forEach(add);
+  room.localParticipant.trackPublications.forEach(add);
+
+  room.participants.forEach((participant: any) => {
+    add(participant);
+    participant.tracks.forEach(add);
+  });
+  sendRoomEvents(room);
+
+  return {
+    result: serializeRoom(room)
+  };
+}
+
+/**
+ * Create a {@link LocalTrack}.
+ * @param {Array<*>} args
+ * @returns {Promise<object>}
+ */
+export async function createLocalTrack(args: any): Promise<any> {
```

Comment:
I think we should consider a more-specific type than `any` here if we are going to do more with these `args` than just `apply` them to a twilio-video.js function.